tasks/main.yml

```diff
@@ -1,34 +1,35 @@
 ---
 
 - name: Get checksums from github.com
-  set_fact:
+  ansible.builtin.set_fact:
```

Comment:
I would prefer to use a `.ansible-lint` instead like https://github.com/systemli/ansible-drift/blob/main/.ansible-lint

Reply:
Try to fix this independently from this PR. See #26